### DISCUSSION TITLE
Multiplayer on offline games

### DISCUSSION THREAD

**Discussion by Allismeaningless** [OnionUI/Onion#1122] [Ideas]
Apologies I am not technical no have no idea if this even is possible. It would be fantastic for 2-4 people with a Miyoo on the same WIFI system to be able to join in each others games. These games are pre-internet and usually 2nd player join in on the controller or keyboard. 

Is it possible to have multiplayer, via a screen share system (one Miyoo hosting the game( similar to how the PlayStation second screen works? Honestly that would be huge but appreciate its probably extremely complicated, if impossible, to implement.

**Comment by Mrluck23** [comment]
Multiplayer in offline games is an interesting concept. I'd like to clarify that traditionally, "offline" games are played without an internet connection, so multiplayer functionality might not work as expected. However, some games do offer local multiplayer modes where players can compete or cooperate on the same device, allowing for fun interactions even without an internet connection. It's important to check the specific game's features to determine if and how multiplayer works in offline scenarios.

**Comment by Allismeaningless** [comment]
I think all the games where it would be a great addition are ones where there was a second controller/joystick or 4 people huddled around an Amiga keyboard. These all predate any kind of internet so it wouldn't be a case of functionality within the game itself, but creating functionality within the Miyoo.

The way i see it working (in my non technical mind) is a WIFI relay between 2/3/4 Minis. One acts as host and plays the game. It casts the game screen to the other minis. The other minis can connect via transmission of their button presses mapped as secondary controllers. 

As I said, no idea if this is possible. But would be great to play some of the arcade classics such with 2 or more minis together.

[edit] i closed this because i thought this was part of the netcode already done. Then i read the instructions :)
